cpp/oneapi/dal/detail/common.cpp

```diff
@@ -52,6 +52,10 @@ template struct ONEDAL_EXPORT integer_overflow_ops<std::uint16_t>;
 template struct ONEDAL_EXPORT integer_overflow_ops<std::uint32_t>;
 template struct ONEDAL_EXPORT integer_overflow_ops<std::uint64_t>;
 
+#if defined(__APPLE__)
+template struct ONEDAL_EXPORT integer_overflow_ops<std::size_t>;
+#endif
+
```

Comment:
Why should we define it for v1? This is preserved only for backward compatibility, all further modifications must be done in the latest vX